### ISSUE TITLE
deno test --coverage: don't print untested lines

### ISSUE THREAD

**Issue by gastrodon** [denoland/deno#9162]
Is there a way to use `deno test --coverage` to get file coverage, without printing every uncovered line to stdout?

Even better, is there a way to write coverage to a file that can be understood by something like [codecov](https://codecov.io)

**Comment by caspervonb** [comment]
Working on improving the output filtering and output formats.

**Comment by gastrodon** [comment]
Yeah, that would be about reasonable. For example with `go test -cover`
```
ok  	git.gastrodon.io/imonke/monkebase	33.621s	coverage: 95.0% of statements in ./...
```

It might also be useful if this could produce a coverage report that's usable by tools like codecov.io

**Comment by caspervonb** [comment]
> It might also be useful if this could produce a coverage report that's usable by tools like codecov.io

Got lcov incoming for 1.8

**Comment by caspervonb** [comment]
I'll look into adding a --summary option once the new rewrite lands which should be fairly soon.

**Comment by GCSBOSS** [comment]
Any chance this has already been released? We do have lcov file generation under `--unstable`.

**Comment by caspervonb** [comment]
Lcov landed for for 1.8.
Rewrote the test runner completely for 1.9.
I'll see if i can get the terse formatter into 1.10, some bugs and technical dept to clean up in there.

**Comment by dsherret** [comment]
> Even better, is there a way to write coverage to a file that can be understood by something like codecov

This need seems to have been met by lcov reports. For details, see https://deno.land/manual/testing#test-coverage
